extensions/applicationinsights-properties-js/src/PropertiesPlugin.ts

```diff
@@ -36,22 +36,27 @@ export default class PropertiesPlugin implements ITelemetryPlugin, ITelemetryCon
     private _nextPlugin: ITelemetryPlugin;
     private _extensionConfig: ITelemetryConfig;
 
-    initialize(config: IConfiguration, core: IAppInsightsCore, extensions: IPlugin[]) {
-        let extensionConfig = config.extensionConfig &&
-            config.extensionConfig[this.identifier] ?
-            config.extensionConfig[this.identifier] : {};
+    public static getDefaultConfig(): ITelemetryConfig {
+        const defaultConfig: ITelemetryConfig = {
+            instrumentationKey: undefined,
```

Comment:
are these truly the defaults for current PROD sdk (1.0.20)?

Reply:
nope. pushed an update now